code/backend/pages/04_Configuration.py

```diff
@@ -340,5 +341,21 @@ def validate_documents():
         ConfigHelper.save_config_as_active(current_config)
         st.success("Configuration saved successfully!")
 
+    with st.popover(":red[Reset confiiguration to defaults]"):
+        st.write(
+            "**Resetting the configuration cannot be reversed, proceed with caution!**"
+        )
+        name = st.text_input('Enter "reset" to proceed', key="reset_configuration")
+        if st.button(
+            ":red[Reset]", disabled=st.session_state["reset_configuration"] != "reset"
+        ):
+            try:
+                ConfigHelper.delete_config()
+            except ResourceNotFoundError:
+                pass
```

Comment:
We might want to surface this error up to the user. If resetting the configuration failed

Reply:
I see now that this is only when the resource in the lob storage isn't found. Still it might make sense to let the user know something unexpected happened

Reply:
Yes, that sounds good!